app/interactors/schedule/create_interactor.rb

```diff
@@ -24,7 +24,7 @@ def find_and_lock_edition
   end
 
   def check_for_issues
-    unless edition.schedulable?
+    unless edition.editable? && edition.scheduled_publishing_datetime.present?
```

Comment:
I think we'd need something here that checks the date is correct - though could be considered part of https://trello.com/c/a4sDMtqD/844-ensure-publishers-meet-requirements-before-they-schedule

Reply:
Agreed it's a bit unusual - perhaps a full-blown requirements checker for this particular action would be the way to go here? I guess we want to force users down the path to success, even though submitting with a date in the past doesn't actually cause an error?!

Reply:
Let's cover this when we chat about error stuff tomorrow. We can start by looking at whether we now know everything we need for the requirements card.